perf/istio-install/create_cluster.sh

```diff
@@ -165,29 +165,29 @@ export CLUSTER_NAME
 mkdir -p "${WD}/tmp/${CLUSTER_NAME}"
 "${WD}/create_sa.sh" "${GCP_SA}" "${GCP_CTL_SA}"
 
+# set config use kpt then create cluster with anthoscli
+function install_with_anthoscli() {
+  kpt cfg set gke_cluster_resources gcloud.core.project ${PROJECT_ID}
```

Comment:
Just to confirm -- are all these env variables guaranteed to be set? If not, I believe this kpt call will erase the default.

Reply:
Also, I'd add doublequotes to all these env variables in case someone passes one with spaces.

Reply:
yes, all these env variables are preset in the previous step.

ok I added double quotes